Create supportability.md

taylorrees1 opened the pull request:
Created a page for supportability in Oracle Database at azure

taylorrees1 committed:
Create supportability.md

Created a page for supportability in Oracle Database at azure

taylorrees1 committed:
Update supportability.md

updated ms.topic

jjaygbay1 commented on articles/oracle/oracle-db/supportability.md:
```diff
@@ -0,0 +1,46 @@
+---
+
+# Supportability - Oracle Database@Azure 
+
```
Let's add a short intro paragraph here about the two-party support system.

jjaygbay1 commented on articles/oracle/oracle-db/supportability.md:
```diff
@@ -0,0 +1,46 @@
+- Bastion and virtual machine (VM) issues including database host connection, software installation, latency, and host performance.
+- VM metrics, database logs, database events.
+
+See [Contact Microsoft Azure Support](https://support.microsoft.com/topic/contact-microsoft-azure-support-2315e669-8b1f-493b-5fb1-d88a8736ffe4) in the Azure documentation for information on Azure support. For SLA information about the service offering, please refer to the [Oracle PaaS and IaaS Public Cloud Services Pillar Document](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2 ...
```
delete 'please refer to the' and use 'see,'

jjaygbay1 reviewed (changes requested):
Thank you. I made two minor edits.

taylorrees1 committed:
Update supportability.md

added changes based on review

-included intro paragraph
-replaced please refer to with see

taylorrees1 committed:
added support file to the ToC

taylorrees1 committed:
added support file to the ToC round 2
